php/PhpJsonApiSamples/Default.html

```diff
@@ -31,41 +31,45 @@
 	<form id="form1" runat="server">
 	<table style="width:40%">
 		<tr><th>Select any option </th></tr>
-		<!--<tr> 
-			<td><a href="ClientSideEncryptionDemo.aspx">ClientSide Encryption </a> </td>
-		</tr>   -->
+		<tr> 
+                        <!-- Demo1 checks the client side validations and then submit the valid form with PayTrace client side library  -->
+			<td><a href="ClientSideEncryptionDemo1.html">Client Side Encryption Demo 1 </a> </td>
+		</tr>   
+                <tr> 
+			 <!-- Demo2 submits the form with PayTrace client side library as soon as form is submitted. -->
+                        <td><a href="ClientSideEncryptionDemo2.html">Client Side Encryption Demo 2 </a> </td>
+		</tr> 
 		<tr>
-                    <td><a href="OAuthTokenJson.php">Authentication Token</a> </td>
+                        <td><a href="OAuthTokenJson.php">Authentication Token</a> </td>
   		</tr>
 		<tr>
 			<td><a href="KeyedSaleJson.php">Keyed Sale (Card-Not Present)</a> </td>
   		</tr>
 		<tr>
 			<td><a href="SwipedSaleJson.php">Swiped Sale (Card Present)</a> </td>
- 		 </tr>
+ 		</tr>
 		<tr>
+                        <td><a href="VaultSaleByCustomerIDJson.php">Vault Sale By Customer ID</a> </td>
+  		</tr>
+                <tr>
 			<td><a href="KeyedAuthorizationJson.php">Keyed Authorization(Card-Not Present)</a> </td>
   		</tr>
 		<tr>
 			<td><a href="CaptureJsonUI.php">Capture Authorized Transaction</a> </td>
   		</tr>
 		<tr>
 			<td><a href="KeyedRefundJson.php">Keyed Refund</a> </td>
- 		 </tr>
+ 		</tr>
 		<tr>
 			<td><a href="VoidTransactionJsonUI.php">Void Transaction</a> </td>
   		</tr>
 		<tr>
-			<td><a href="CreateCustomerJson.php">Create Customer Profile</a> </td>
+			<td><a href="CreateCustomerJson.php">Create Customer Profile (Tokenization)</a> </td>
  		</tr> 
- 		<tr>
-    		<td><a href="VaultSaleByCustomerIDJson.php">Vault Sale By Customer ID</a> </td>
-  		</tr>
+ 		
```

Comment:
Yeah I think providing some more info about client side encryption demo 1 and demo 2 would be helpful.

Thanks,

RUPA MODHA  DEVELOPER SUPPORT ENGINEER
PayTrace The Secure Advantage

T 509 228 8033<applewebdata://A56E5FD9-6AB4-451E-82A4-B5A8816D73C2/tel/%20360%20635%206291> · F 360 326 1985<applewebdata://A56E5FD9-6AB4-451E-82A4-B5A8816D73C2/tel/%20360%20326%201985>
Rmodha@PayTrace.com<mailto:Joel@PayTrace.com>· https://paytrace.com<https://paytrace.com/>
Twitter @PayTrace<https://twitter.com/PayTrace> · LinkedIn PayTrace<http://www.linkedin.com/company/paytrace>





From: quachj <notifications@github.com<mailto:notifications@github.com>>
Reply-To: PayTrace/api-demo <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, June 30, 2016 at 11:31 AM
To: PayTrace/api-demo <api-demo@noreply.github.com<mailto:api-demo@noreply.github.com>>
Cc: Rupa Modha <rmodha@paytrace.com<mailto:rmodha@paytrace.com>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [PayTrace/api-demo] Feature/api 60and61 (#7)


In php/PhpJsonApiSamples/Default.html<https://github.com/PayTrace/api-demo/pull/7#discussion_r69185821>:

>               </tr>
> -             <tr>
> -             <td><a href="VaultSaleByCustomerIDJson.php">Vault Sale By Customer ID</a> </td>
> -             </tr>
> +


The verbiage for these links seem more self explanatory so not sure if you need a description. Just first two client encryption seem a little confusing of what they actually are for. ex demo1 and demo2 doesn't really say what they do.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/PayTrace/api-demo/pull/7/files/3e62dae1fe3faa0c223cb839c0461f26c4f51be0#r69185821>, or mute the thread<https://github.com/notifications/unsubscribe/APErkdhLLE72d1dRS6z_MJ5w-PEj5AzPks5qRAuCgaJpZM4JCa-c>.